jboss-kie-kieserver/tests/bats/jboss-kie-kieserver.bats

```diff
@@ -617,3 +619,34 @@ teardown() {
     echo "GC_MAX_METASPACE_SIZE=${GC_MAX_METASPACE_SIZE}"
     [[ "${GC_MAX_METASPACE_SIZE}" == "${KIE_SERVER_MAX_METASPACE_SIZE}" ]]
 }
+
+
+@test "Verify if the jbpm cache is contained in the standalone-openshift.xml when KIE_SERVER_JBPM_CLUSTER is true" {
+  export KIE_SERVER_JBPM_CLUSTER="true"
+  if grep -q '<cache-container name="jbpm">' ${JBOSS_HOME}/standalone/configuration/standalone-openshift.xml  ; then
+    false
+  else
+    true
+  fi
+  configure_jbpm_cluster
+  if grep -q '<cache-container name="jbpm">' ${JBOSS_HOME}/standalone/configuration/standalone-openshift.xml  ; then
```

Comment:
why add it twice?
can't you use xmllint to check if the content was added on the xml istead use these ifs?

Reply:
Done